scu-de/src/fiskaltrust.Middleware.SCU.DE.CryptoVision/Interop/File/CryptoVisionFileProxy.cs

```diff
@@ -427,6 +427,20 @@ public async Task<SeResult> SeInitializePinsAsync(byte[] adminPuk, byte[] adminP
             });
         }
 
+        /// <summary>
+        /// This is the overload of the InitializePinAsync command which is used ONLY with V2 hardware
+        /// </summary>
+        public async Task<SeResult> SeInitializePinsAsync(string userId, byte[] userPuk)
```

Comment:
Looks like currently we are not calling this from our CryptoVisionSCU. We should add this call for V2 based TSEs